src/content/release/breaking-changes/deep-links-flag-change.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Deep links flag change
+
+---
+
+## Summary
+
+The default value for Flutter’s deep linking flag has changed from false to true, meaning deep linking is now opt-in by default.  
+
+## Migration guide
+
+
+If you’re using Flutter’s default deep linking setup, this is not a breaking change for you. 
+
+However, if you’re using third-party plugins for deep links, such as:
+
+[firebase dynamic links][https://firebase.google.com/docs/dynamic-links]
+[uni_link][https://pub.dev/packages/uni_links]
+[app_links][https://pub.dev/packages/app_links]
+This update will introduce a breaking change. In this case, you’ll need to manually reset the Flutter deep linking flag to false.
+
+Before: non-op
+
+After:
+
+Manually disable the deep linking flag if you're using third-party plugins:
+
+* Android Manifest file
+
+```dart
```

Comment:
@hannah-hyj, I _believe_ the build is failing because you label this code as `dart`, when it's `yaml`. Can you fix this and also add "yaml" to the iOS example below. If I'm right, this should fix the website build.